.github/workflows/jira-ci.yml

```
@@ -15,17 +15,20 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: jira-pull-requests
     steps:
+      - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
```

```suggestion
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
```
и так во всех местах